packages/create-plugin/src/utils/utils.config.ts

```diff
@@ -23,10 +23,10 @@ export function getConfig(): CreatePluginConfig {
     return {
       ...rootConfig,
       ...userConfig,
-      version: rootConfig.version,
+      version: rootConfig!.version,
       features: createFeatureFlags({
-        ...rootConfig.features,
-        ...userConfig.features,
+        ...rootConfig!.features,
+        ...userConfig!.features,
```

Comment:
Aaa, these are necessary because of the `tsconfig` updates!

Reply:
Yep! We're kind of abusing the try/catch here to figure out what to return so I thought it was _okay_ to do?

Reply:
I think it is 👍